security/authorization/compiler/api/compiler.go

```diff
@@ -0,0 +1,145 @@
+// +build go1.10
```

Comment:
go 1.9 tests won't be run anymore. Now you should be able to remove this tag.

Reply:
removed for all files